packages/@react-aria/virtualizer/src/ScrollView.tsx

```diff
@@ -128,8 +129,9 @@ function ScrollView(props: ScrollViewProps, ref: RefObject<HTMLDivElement>) {
       return;
     }
 
-    let w = dom.clientWidth;
-    let h = dom.clientHeight;
+    let computedStyles = getComputedStyle(dom);
+    let w = parseFloat(computedStyles.width) + parseFloat(computedStyles.paddingLeft) + parseFloat(computedStyles.paddingRight);
+    let h = parseFloat(computedStyles.height) + parseFloat(computedStyles.paddingTop) + parseFloat(computedStyles.paddingBottom);
```

Comment:
As mentioned in https://github.com/adobe/react-spectrum/pull/4846/files#r1279884044, the core issue was that the TableView's column head wrapper would occasionally be wider than the actual TableBody due to `clientWidth`'s pixel rounding behavior. This would result in the window/flex container around the TableView rendering scrollbars if the TableView's width was matching the exact width of said window/flex container -> resulting in the TableView resizing itself to match the new (smaller) available width of the container -> a new smaller visibleRect is calculated and applied to the TableView's header width causing the scrollbars to disappear -> cycle begins again